wallet-server-ui/src/styles.scss

```diff
@@ -312,7 +312,7 @@ mat-form-field {
 .execute-container {
```

Comment:
Please remove changes to this file.

Reply:
Resolved by 24e0bf481e989e782838ebfeccbb1400a717ddae